agent/handshake/agency.go

```diff
@@ -86,13 +86,12 @@ func AnchorAgent(email string) (agent *cloud.Agent, err error) {
 	assert.P.True(!walletAlreadyEsists, "wallet cannot exist when onboarding")
 	agent.OpenWallet(*aw)
 
-	// Bind pairwise Steward <-> New Agent
-	stewardDID := steward.RootDid()
-
-	// Promote new agent by Trusted Anchor DID
-	anchorDid := agent.CreateDID("")
-	err2.Check(steward.SendNYM(anchorDid, stewardDID.Did(),
-		findy.NullString, "TRUST_ANCHOR"))
+	anchorDid := agent.CreateDID(seed)
+	if steward != nil {
+		// Promote new agent by Trusted Anchor DID if steward is available
```

Comment:
You said this in the meeting by yourself. Maybe put `assert.P.True(seed == "")` in side `if steward != nil ` block?